_posts/2018-07-02-blockchain-sharing-economy-autonomes-fahren.md

```diff
@@ -0,0 +1,57 @@
+---
+layout:     post
+title:      Lean Startup oder Was haben Blockchain, Sharing Economy und autonomes Fahren gemeinsam?
+authorPage: Christiane Gerigk
+published:  02.07.2018
+image:      Universal_Tile_XCamp18_Thomas-Edison-700x441.png
```

Comment:
The image is missing in the PR. It should be added to the `media` folder, so it should be specified as `media/Universal_Tile_XCamp18_Thomas-Edison-700x441.png` here.
(This is new behavior after my re-structuring)

Reply:
I've merged your other PR into this one, so the image is included now.